### PR TITLE
feat: add summary view

### PR DESCRIPTION
**Pull request by urielPalomares**

### CODE REVIEW

File: components/game/game.tsx [hunk 1]
```diff
@@ -36,6 +36,9 @@ const GameLayout: NextPage = () => {
   };
 
   const handleNextTrack = () => {
+    if (gameState.currentSong + 1 === gameState.playlist.length) {
+      router.push('/summary');
+    }
     dispatch(gameSlicesActions.nextGame());
```
Comment (Andres2D) [inline components/game/game.tsx:42]: Is not neccesary to dispatch `nextGame`at the end.

File: components/summary/summary.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      })
+      return answers.length;
+    };
+    
+    const getIncorrectAnswers = () => {
+      const answers =  gameState.playlist.filter((element) => {
+        return element.state === 'wrong';
+      })
+      return answers.length;
+    };
```
Comment (Andres2D) [inline components/summary/summary.tsx:28]: You just need a single method, the other data could be calculated.

File: components/summary/summary.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        </Box>
+        <Box flex='1' m={2} color='tomato'>
+          <Heading size="xl">
+            Incorrect
+          </Heading>
+          <Heading size="lg"
+          display='flex' 
+          flexDirection='column' 
+          alignItems='center'>
+            {getIncorrectAnswers()}
```
Comment (Andres2D) [inline components/summary/summary.tsx:61]: Here could be the calculation, like: { game.playlist.lenght - getCorrectAnswers }

File: pages/summary/index.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    <>
+      <Metadata title={`Summary`}/>
+      <SummaryLayout />
+    </>
+  );
+};
+
+export const getServerSideProps = async(context: any) => {
+  const session = await getSession({req: context.req});
+  let playlist = await getPlaylistGame(session?.accessToken, Number(process.env.LIMIT_SONGS) || 10);
```
Comment (Andres2D) [inline pages/summary/index.tsx:28]: You don't need the playlist data here.

File: pages/summary/index.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+        destination: '/auth',
+        permanent: false
+      }
+    }
+  }
+  
+  return {
+    props: { 
+      session,
+      playlist
```
Comment (Andres2D) [inline pages/summary/index.tsx:41]: Don't need to send the data to the component.

**Review by Andres2D — CHANGES_REQUESTED** [review]

File: components/summary/summary.tsx (hunk 3 above)
Comment (urielPalomares) [reply components/summary/summary.tsx]: Ready Sir

File: components/summary/summary.tsx (hunk 2 above)
Comment (urielPalomares) [reply components/summary/summary.tsx]: Ready Sir

File: components/game/game.tsx (hunk 1 above)
Comment (urielPalomares) [reply components/game/game.tsx:42]: Ready Sir

**Review by Andres2D — APPROVED** [review]

**Pull request merged**